bitnami/rabbitmq-cluster-operator/README.md

```diff
@@ -0,0 +1,315 @@
+
+
+### Other Parameters
+
+| Name                                       | Description                                                                 | Value                    |
+| ------------------------------------------ | --------------------------------------------------------------------------- | ------------------------ |
+| `rbac.create`                              | Specifies whether RBAC resources should be created                          | `true`                   |
+| `serviceAccount.create`                    | Specifies whether a ServiceAccount should be created                        | `true`                   |
+| `serviceAccount.name`                      | The name of the ServiceAccount to use.                                      | `""`                     |
+| `metrics.enabled`                          | Create a service for accessing the metrics endpoint                         | `false`                  |
+| `metrics.service.type`                     | RabbitMQ Cluster Operator metrics service type                              | `ClusterIP`              |
+| `metrics.service.port`                     | RabbitMQ Cluster Operator metrics service HTTP port                         | `80`                     |
+| `metrics.service.nodePorts.http`           | Node port for HTTP                                                          | `""`                     |
+| `metrics.service.clusterIP`                | RabbitMQ Cluster Operator metrics service Cluster IP                        | `""`                     |
+| `metrics.service.loadBalancerIP`           | RabbitMQ Cluster Operator metrics service Load Balancer IP                  | `""`                     |
+| `metrics.service.loadBalancerSourceRanges` | RabbitMQ Cluster Operator metrics service Load Balancer sources             | `[]`                     |
+| `metrics.service.externalTrafficPolicy`    | RabbitMQ Cluster Operator metrics service external traffic policy           | `Cluster`                |
+| `metrics.service.annotations`              | Additional custom annotations for RabbitMQ Cluster Operator metrics service | `{}`                     |
+| `metrics.serviceMonitor.enabled`           | Specify if a servicemonitor will be deployed for prometheus-operator        | `false`                  |
+| `metrics.serviceMonitor.jobLabel`          | Specify the jobLabel to use for the prometheus-operator                     | `app.kubernetes.io/name` |
+| `metrics.serviceMonitor.interval`          | Scrape interval. If not set, the Prometheus default scrape interval is used | `""`                     |
+| `metrics.serviceMonitor.metricRelabelings` | Specify additional relabeling of metrics                                    | `[]`                     |
+| `metrics.serviceMonitor.relabelings`       | Specify general relabeling                                                  | `[]`                     |
```

Comment:
Please re-run the `readme-generator` tool, the metrics parameters should have its own section.

Reply:
Done